.changeset/long-suns-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/providers": minor
```

Comment:
Is this a breaking change?

Reply:
I don't think so because your fix took into uses the proper Receipts type. Also, all receipts properties are still included into the other queries responses.

Reply:
Should this be a `patch` then?

Reply:
Makes sense